app/models/host/managed.rb

```diff
@@ -507,7 +508,7 @@ def self.registered_provision_methods
   end
 
   def self.valid_rebuild_only_values
-    Nic::Managed.rebuild_methods.values + Host::Managed.rebuild_methods.values
+    Nic::Managed.rebuild_methods.values + Host::Managed.rebuild_methods.values if Host::Managed.respond_to?(:rebuild_methods)
```

Comment:
not sure if this is on purpose or not, but the if clause here applies to the whole line. Perhaps `return nil` early instead to make it clearer this is intentional?
does it make sense to only return `Nic::Managed.rebuild_methods.values` if `unattended=false` or not? since orchestration is included in Nic::Managed always, that method will work.